.clang-format

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
We could copy'n'paste the clang-format CI check into this repo, to have formatting validated right along with adding this file :)